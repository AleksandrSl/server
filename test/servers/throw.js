#!/usr/bin/env node

var Server = require('../../server')
new Server({
  env: 'test',
  uniqName: 'server',
  subprotocol: [1, 0],
  supports: [1]
})

setTimeout(function () {
  var error = new Error('Test Error')
  error.stack = error.stack.split('\n').slice(0, 2).join('\n') + '\n' +
    '    at ontimeout (timers.js:365:14)\n' +
    '    at tryOnTimeout (timers.js:237:5)\n' +
    '    at Timer.listOnTimeout (timers.js:207:5)\n'
  throw error
}, 10)
