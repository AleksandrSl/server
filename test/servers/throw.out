
 ERROR  1970-01-01 00:00:00 Test Error
        at Timeout._onTimeout (test/servers/throw.js:12:15)
        at ontimeout (timers.js:365:14)
        at tryOnTimeout (timers.js:237:5)
        at Timer.listOnTimeout (timers.js:207:5)
        

 INFO  1970-01-01 00:00:00 Shutting down Logux server
